D_transl: 800
K_transl: [500,500,500]
D_orient: [60,60,60]
